books/free-programming-books-id.md

```diff
@@ -146,7 +146,7 @@
 
 ### NoSQL
 
-* [MongoDB Untuk Indonesia](https://kristories.gitbooks.io/pengantar-mongodb/content/)
+* [MongoDB Untuk Indonesia: Memahami Konsep dan Implementasi MongoDB](https://kristories.gumroad.com/l/mongodb-untuk-indonesia) - Wahyu Kristianto (PDF)
```

Comment:
Thanks for the update - for gumroad, you'll need to add the `(email address *requested*, not required)` annotation. This is thae same model as syncfusion books.